enclave_wrangler/utils.py

```diff
@@ -157,30 +157,38 @@ def response_failed(response: Response) -> bool:
 
 
 def handle_response_error(
-    response: Response, error_dir: Union[str, None] = None, fail: bool = False, calling_func: str = 'response error'
+    response: Response, error_dir: Union[str, None] = None, calling_func: str = 'response error'
 ):
     """This code was taken out of make_objects_request() (formerly get_objects_by_type()). Trying to use it
-    for all Response errors now. Not sure if it's entirely appropriate."""
+    for all Response errors now. Not sure if it's entirely appropriate.
+
+    TODO: Test this! Siggie and Hope made changes in late June and we didn't test them. If this breaks, blame them
```

Comment:
Just commenting that I've seen these changes to error reporting! Fine with me. Hope it works. If you'd like, you can prioritize adding a test for this ahead of #464. I leave it up to you.